### ISSUE TITLE
accessToken is undefined and roles are also not coming in auth  

### ISSUE THREAD

**Issue by Mohsinfistix** [aerogear/keycloak-connect-graphql#134]
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.isAuthorizedByRole = void 0;
const KeycloakContext_1 = require("../KeycloakContext");
/**
 *
 * @param roles the list of roles the user should match against
 * @param context the graphql context that contains the user info
 */
function isAuthorizedByRole(roles, context) {
    if (!(context && context[KeycloakContext_1.CONTEXT_KEY])) {
        console.error(`context.${KeycloakContext_1.CONTEXT_KEY} is missing. Keycloak integration is probably misconfigured`);
        return false;
    }
    for (const role of roles) {
        if (context[KeycloakContext_1.CONTEXT_KEY].hasRole(role)) {
            return true;
        }
    }
    return false;
}
exports.isAuthorizedByRole = isAuthorizedByRole;

**Comment by BigBallard** [comment]
@Mohsinfistix you need to provide more context and explanation than just posting code if you expect any help.

**Comment by proftom** [comment]
Agreed @Mohsinfistix needs to provide more context, however, I want to ask, if I passes a Bearer token in an authorization header, should I expect accessToken to equal that bearer token?

**Comment by BigBallard** [comment]
@proftom that is my understanding. I haven not checked myself but would be easy enough to test in isolation.

**Comment by okanji** [comment]
I am having a similar issue, the access token is not defined but I do have a bearer token in the header

![image](https://user-images.githubusercontent.com/30703259/163740158-f20d48ff-4010-43bb-8fbe-d08681504cd0.png)
